src/index.js

```diff
@@ -271,21 +300,35 @@ IdTokenVerifier.prototype.decode = function (token) {
  * of the previously verified id_token.
  * @param {validateAccessTokenCallback} cb callback used to notify the results of the validation.
  */
-IdTokenVerifier.prototype.validateAccessToken = function (accessToken, alg, atHash, cb) {
+IdTokenVerifier.prototype.validateAccessToken = function(
+  accessToken,
+  alg,
+  atHash,
+  cb
+) {
   if (this.expectedAlg !== alg) {
-    return cb(new error.TokenValidationError('Algorithm ' + alg +
-      ' is not supported. (Expected alg: ' + this.expectedAlg + ')'));
+    return cb(
+      new error.TokenValidationError(
+        'Algorithm ' +
```

Comment:
If we're using Babel/ES6 transforms here maybe use string interpolation?  

```js
`Algorithm ${alg} is not supported. (Expected alg: ${this.expectedAlg})`
```

Reply:
I didn't convert the code to ES6 and have no plans to convert everything now. I only converted how we export/import modules.

I used https://github.com/lebab/lebab to convert es5 to es6 in a few projects and we can do a conversion later. For now, I'm only converting modules.